pages/bibliographic-references.xml

```diff
@@ -17,113 +17,140 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-    does not yet have a unique tag, please assign one with this format
-    and using only Latin alphabet letters</p>
-         <cit>
-            <quote>
- bm:AuthorYYYYkeyword
-</quote>
-         </cit>
-            <p>Please make sure this contains <hi rendition="simple:bold">no special characters</hi>, only digits
-               and Latin ASCII characters; no dashes/hyphens/underscores/diacritics. Make the tag as short as possible. To check that this is unique, after saving the
-    record in Zotero, click on the tag and check that only one item has
-    this tag. If not, change the keyword of the tag just assigned.</p>
-         <p>When adding records to the Zotero Group Library, please follow
-    the principles <ref target="zotero">at this
-    page</ref>.</p>
-</div>
-<div type="level2">
-  <p>You can use a <att>corresp</att> to link a <gi>bibl</gi> element to a
-part of the file with an <att>xml:id</att>.</p>
-<egXML xmlns="http://www.tei-c.org/ns/Examples">
-  <bibl corresp="#t3arabic">
-  <ptr target="bm:Enger1856Transitu"/>
-  </bibl>
-</egXML>
-<p>The above example would point the bibliographical reference directly to the <gi>title</gi> with that ID.</p>
-<egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <head>Bibliographic references for enriched records</head>
+            <p>Following <ref target="https://github.com/BetaMasaheft/Documentation/issues/2630">issue
+               2630</ref>, in cases of enriching a historical catalogue description, such as verifying images or
+               adding new information, to acknowledge this, one can include the marked-up by a
+                  <gi>bibl</gi> element containing a <gi>ptr</gi> element with a <att>target</att>,
+               with <tag>bm:BmWebsite</tag>, as in the following example: <egXML
+                  xmlns="http://www.tei-c.org/ns/Examples">
+                  <listBibl type="catalogue">
+                     <bibl>
+                        <ptr target="bm:BmWebsite"/>
+                     </bibl>
+                  </listBibl>
+               </egXML>
+            </p>
+            <p> As a result, instead of the standard note saying 'This manuscript description is
+               based on the catalogues listed in the catalogue bibliography', the record will
+               contain the following note: 'This manuscript description is based on the catalogue(s)
+               listed in the catalogue bibliography and has been corrected or enriched by the Bm
+               project team; please check Revision history for more information' </p>
 
-                 <bibl corresp="#blorient481">
-                    <ptr target="bm:Wright1877BM"/>
-                    <citedRange unit="page">2-4</citedRange>
-                 </bibl>
-</egXML>
+            <p>Note that for newly catalogued manuscripts, no additional action is required, and
+               the reference <gi>bibl</gi><ptr target="bm:BmWebsite"/> should not be added.</p>
```

Comment:
I think here the better markup would be `<tag><bibl><ptr target="bm:BmWebsite"/></bibl></tag>` for the entire tag

Reply:
> I think here the better markup would be `<tag><bibl><ptr target="bm:BmWebsite"/></bibl></tag>` for the entire tag

the schema does not allow it.